gfx.h

```diff
@@ -191,15 +191,23 @@ GfxRaytracingPrimitive const *gfxAccelerationStructureGetRaytracingPrimitives(Gf
 
 class GfxRaytracingPrimitive { GFX_INTERNAL_NAMED_HANDLE(GfxRaytracingPrimitive); public: };
 
+enum GfxRaytracingPrimitiveFlags
+{
+    GfxRaytracingPrimitiveFlags_None = 0,
+    GfxRaytracingPrimitiveFlags_NonOpaque = 1,
+
+    GfxRaytracingPrimitiveFlags_Count
+};
```

Comment:
Code convention is to start the constant name with a `k` here.

Also, what about keeping the flags defined the same way than say the `GfxCreateContextFlag` enum for instance?
So something more like this:
```
enum GfxBuildRaytracingPrimitiveFlag
{
    kGfxBuildRaytracingPrimitiveFlag_Opaque = 1 << 0
};
```

Therefore, if you do not pass the `kGfxBuildRaytracingPrimitiveFlag_Opaque` bit flag, the BVH is considered non-opaque.
Will allow to add possibly more flags later on.

Reply:
Yep that makes sense. I had based the enum on `GfxCpuAccess` which is why it was the way it was.